### PR TITLE
'Added a new data structure in cpp'

### PR DESCRIPTION
**Pull request by suraj0223**
Trie data structure is added in cpp/ds

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -458,6 +458,15 @@
       "contributions": [
         "code"
       ]
+    },
+    {
```
Comment (iamrajiv) [inline .all-contributorsrc:462]: @suraj0223 remove unnecessary commit and files

**Review by iamrajiv — CHANGES_REQUESTED** [review]

**Comment by suraj0223** [comment]
Unable to remove commit : 6abd190

**Comment by iamrajiv** [comment]
@suraj0223 I am not able to edit the files. Please, enable `Allow edits from maintainers`.

**Comment by suraj0223** [comment]
Hey @iamrajiv  there is some issue with my forked repo. Could I create another PR for the same.

**Comment by iamrajiv** [comment]
@suraj0223 Sure.

**Pull request closed without merging**